Code/Tools/ProjectManager/Source/GemCatalog/GemModel.cpp

```diff
@@ -764,13 +847,84 @@ namespace O3DE::ProjectManager
         const auto& versions = GemModel::GetGemVersions(modelIndex);
         if (versions.count() < 2)
         {
+            // there is only one version available
             return false;
         }
 
         auto currentVersion = modelIndex.data(RoleVersion).toString();
+        if (compatibleOnly)
+        {
+            // versions are ordered from highest to lowest
+            for (auto itr = versions.cbegin(); itr != versions.cend(); itr++)
+            {
+                const GemInfo& versionGemInfo = itr->value<GemInfo>();
+                if (versionGemInfo.IsCompatible())
+                {
+                    if (currentVersion != versionGemInfo.m_version)
+                    {
+                        return true;
+                    }
+
+                    // if this is a remote gem, show that the update is available if we
+                    // haven't downloaded it yet and the user has downloaded an older version
+                    if (gemInfo.m_gemOrigin == GemInfo::Remote && gemInfo.m_downloadStatus == GemInfo::NotDownloaded)
+                    {
+                        itr++;
+                        while (itr != versions.cend())
+                        {
+                            const GemInfo& olderVersionGemInfo = itr->value<GemInfo>();
+                            if (olderVersionGemInfo.m_version != currentVersion &&
+                                (olderVersionGemInfo.m_downloadStatus == GemInfo::DownloadSuccessful ||
+                                olderVersionGemInfo.m_downloadStatus == GemInfo::Downloaded))
+                            {
+                                // found an older version that was downloaded
+                                return true;
+                            }
+                            itr++;
+                        }
+
+                        // did not find an older version that was downloaded
+                        return false;
+                    }
+
+                    return currentVersion != versionGemInfo.m_version;
+                }
+            }
+            return false;
+        }
+        else
+        {
+            if (currentVersion != versions.at(0).value<GemInfo>().m_version)
+            {
+                return true;
+            }
+
+            // if this is a remote gem that hasn't been downloaded, show that the update is
+            // available the user has downloaded an older version
```

Comment:
There seems to be a missing conjunction `to` here?
```suggestion
            // if this is a remote gem that hasn't been downloaded, show that the update is
            // available to the user has downloaded an older version
```